### ISSUE TITLE
False-positive Overriding WordPress globals is prohibited message

### ISSUE THREAD

**Issue by david-binda** [Automattic/VIP-Coding-Standards#9]
I have noticed that when using the WordPressVIPMinimum standard I'm getting a false positive ERROR "Overriding WordPress globals is prohibited" when checking following code:

```
$ cat test11.php
<?php

function a(){
	global $post_type;
}

function b() {
	$post_type = get_post();
}
$ phpcs test11.php --standard=WordPressVIPMinimum

FILE: test11.php
-------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------
 8 | ERROR | Overriding WordPress globals is prohibited
-------------------------------------------------------------------------

Time: 20ms; Memory: 4Mb
```

I'm not getting the error when using the `WordPress-VIP` standard:

```
$ phpcs test11.php --standard=WordPress-VIP

FILE: test11.php
-------------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
-------------------------------------------------------------------------
 3 | ERROR | [x] Expected 1 space before opening brace; found 0
-------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------

Time: 41ms; Memory: 6Mb
```

But am getting the error again when using the `WordPress` standard:

```
$ phpcs test11.php --standard=WordPress

FILE: test11.php
-------------------------------------------------------------------------
FOUND 5 ERRORS AFFECTING 4 LINES
-------------------------------------------------------------------------
 1 | ERROR | [ ] Missing file doc comment
 3 | ERROR | [ ] Missing function doc comment
 3 | ERROR | [x] Expected 1 space before opening brace; found 0
 7 | ERROR | [ ] Missing function doc comment
 8 | ERROR | [ ] Overriding WordPress globals is prohibited
-------------------------------------------------------------------------
PHPCBF CAN FIX THE 1 MARKED SNIFF VIOLATIONS AUTOMATICALLY
-------------------------------------------------------------------------

Time: 124ms; Memory: 6Mb
```

Something is off...

**Comment by justnorris** [comment]
All I'm getting is this:

```
------------------------------------------------------------------------------------------------------------------
FOUND 0 ERRORS AND 2 WARNINGS AFFECTING 2 LINES
------------------------------------------------------------------------------------------------------------------
 4 | WARNING | Unused global variable $post_type. (WordPressVIPMinimum.Variables.VariableAnalysis.UnusedVariable)
 8 | WARNING | Unused variable $post_type. (WordPressVIPMinimum.Variables.VariableAnalysis.UnusedVariable)
------------------------------------------------------------------------------------------------------------------
```

**Comment by rebeccahum** [comment]
I get the same as @justnorris.
